test/function_call_test.js

```diff
@@ -82,19 +82,32 @@ describe('function calls', () => {
     `);
   });
 
-  it.skip('places parentheses in calls with multi-line function arguments after the closing brace', () => {
+  it('places parentheses in calls with multi-line function arguments after the closing brace', () => {
     check(`
       promise.then ->
+        a()
         b # c
       d
     `, `
       promise.then(function() {
+        a();
         return b; // c
       });
       d;
     `);
   });
 
+  it('places parentheses in calls with fat arrow function arguments after the closing brace', () => {
```

Comment:
This description doesn't seem to match the code. There's no fat arrow in the input, and there's no closing brace in the output (because esnext eats it).